src/assets/ArrowDown.tsx

```diff
@@ -0,0 +1,14 @@
+export function FeedSVG() {
```

Comment:
"FeedSVG" is function name that has already been used.
You could use:
`import * as OtherSVG from "./ArrowDown`
But it's not ideal, make sure to make your SVG Components have unique function names.

Reply:
Names have been changed